app/main/views/g_cloud.py

```diff
@@ -58,9 +58,32 @@
 
 @main.route('/g-cloud')
 def index_g_cloud():
+    return redirect(url_for("direct_award_public.pre_project_task_list", framework_family="g-cloud"), 301)
+
+
+@direct_award_public.route("/<string:framework_family>/start-project", methods=("GET",))
```

Comment:
@karlchillmaid any thoughts on this url?

Reply:
Are we doing something similar for DOS framework start pages too? (if so, what?)
Does it need to say '/start-project' – as project is not a term we user with end users (at the moment)?
Would `/start` work on it's own? 🤔